rfb/rfb.h

```diff
@@ -895,6 +895,8 @@ extern rfbBool rfbTightDisableGradient;
 extern int rfbNumCodedRectsTight(rfbClientPtr cl, int x,int y,int w,int h);
 
 extern rfbBool rfbSendRectEncodingTight(rfbClientPtr cl, int x,int y,int w,int h);
+extern rfbBool rfbSendTightHeader(rfbClientPtr cl, int x, int y, int w, int h);
+extern rfbBool rfbSendCompressedDataTight(rfbClientPtr cl, char *buf, int compressedLen);
```

Comment:
I reckon rfbSendTightData would be better naming in line with rfbSendTightHeader.